src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -58,6 +58,14 @@ class Request
     const METHOD_TRACE = 'TRACE';
     const METHOD_CONNECT = 'CONNECT';
 
+    const HEADER_REQUEST_METHOD = 'REQUEST_METHOD';
+    const HEADER_METHOD_OVERRIDE = 'X-HTTP-METHOD-OVERRIDE';
+    const HEADER_CONTENT_TYPE = 'CONTENT_TYPE';
+    const HEADER_REQUEST_URI = 'REQUEST_URI';
+
+    const PORT_HTTPS = 443;
+    const PORT_HTTP = 80;
```

Comment:
all these constants should be removed: adding them would make them part of the public API, which means they would be covered by the BC promise. That's not desired.